README.rst

```diff
@@ -73,6 +73,30 @@ as well (we still run the 'style checks' at least).
 We hope to get to the bottom of this at some point and would be most grateful
 if you have any hint or pointer that may help tracking down this issue.
 
+Migrating from 'legacy hamster'
+---------------------------------
+In case you are wondering "Will I be able to continue to use my 'legacy hamster'
+database with this rewrite?" the answer is "yes and no".
+This new version of hamster significantly raises the standart in terms of data consistency. Unlike
+before there will be no way to have "Facts" that do not have a dedicated "end" or even
+"multiple facts at a given point in time".
+
+There will be a way to import data that still constitutes valid 'facts' (that
```

Comment:
I try not to be a grammar nazi and accept that “data” can be sometimes be uncountable but IMHO when the word can be substituted for a plural word like facts or items, it should be regarded as a plural noun.